backend/app/services/knowledge/knowledge_service.py

```
@@ -728,8 +781,18 @@ def create_document(
         if not kb:
             raise ValueError("Knowledge base not found or access denied")
 
+        # Check permission for organization knowledge base (admin only)
+        if kb.namespace == "organization":
+            from app.models.user import User
+
+            user = db.query(User).filter(User.id == user_id).first()
+            if not user or user.role != "admin":
+                raise ValueError(
+                    "Only admin can add documents to organization knowledge base"
+                )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)